v0.1 YesWiki/src/themes/tools/bazar/templates/carousel_v2.tpl.html

```diff
@@ -46,22 +46,35 @@
 Fade content bs-carousel with hero headers
 Code snippet by maridlcrmn (Follow me on Twitter @maridlcrmn) for Bootsnipp.com
 */
+
+/*.col-sm-5 {
+	 margin-bottom: 1em !important;
+}*/
```

Comment:
```suggestion
```
J’enlèverai bien ce commentaire qui ne semble plus utilisé.

Reply:
en effet ! Je ne voulais pas supprimer cette modif totalement avant d'être sûre que l'affichage était ok, mais si ça semble bon, ce petit post-it n'est plus nécessaire =)